src/software/ai/hl/stp/play/defense_play_test.cpp

```diff
@@ -156,28 +155,23 @@ TEST_F(DefensePlayTest, test_defense_play_close_to_net)
             robotHalt(world_ptr, yield);
             // Attacker in front of enemy with the ball
             Rectangle attacker_rect(Point(-3.5, 1.2), Point(-2.3, 0.2));
-            robotInPolygon(1, attacker_rect, world_ptr, yield);
+            robotInPolygon(attacker_rect, 1, world_ptr, yield);
 
             // The rectangle for the right crease defender and one of the shadowing robots
             // is shared to make the test less brittle since the AI may assign different
             // friendly robots to the to each tactic.
 
             // Two friendly crease defenders should be close to the goalie
             Point goalie_position = world_ptr->friendlyTeam().goalie()->position();
-            Rectangle left_crease_defender_rect(
-                Point(goalie_position.x(), goalie_position.y() + 0.45),
-                Point(goalie_position.x() + 0.55, goalie_position.y()));
-            Rectangle right_crease_defender_and_shadow_rect(
-                Point(goalie_position.x(), goalie_position.y()),
-                Point(goalie_position.x() + 1.0, goalie_position.y() - 0.3));
-            robotInPolygon(2, left_crease_defender_rect, world_ptr, yield);
-            robotInPolygon(3, right_crease_defender_and_shadow_rect, world_ptr, yield);
+            Rectangle left_crease_defender_and_shadow_rec(
+                Point(goalie_position.x() + 1, goalie_position.y() + 0.8),
+                Point(goalie_position.x() + 1.75, goalie_position.y() + 0.4));
+            robotInPolygon(left_crease_defender_and_shadow_rec, 3, world_ptr, yield);
```

Comment:
Any particular reason why the dimensions of the rectangle were changed here?

Reply:
When I reran the test it seemed that the test was failing for that particular left_crease_defender even though the validation functions shouldn't change the behavior. I'm not too sure as to why it would affect the robot positioning.. maybe @jonathanlew can add some insight?

Reply:
I have no idea